src/lib/sixel.c

```diff
@@ -317,11 +317,12 @@ write_rle(int* printed, int color, FILE* fp, int seenrle, unsigned char crle){
 // Closes |fp| on all paths.
 static int
 write_sixel_data(FILE* fp, int leny, int lenx, const sixeltable* stab, int* parse_start){
-  *parse_start = fprintf(fp, "\ePq");
+  // Set P2=1, turning empty pixels transparent
+  *parse_start = fprintf(fp, "\eP0;1;0q");
```

Comment:
ahhhh, so this is why using Set Raster Attributes never worked for me -- it's not the 1 in SRA which does this switch, but the 0;1;0 up here. how'd i miss that? stupidity, nicholas, stupidity is how you missed that :(.

this is great, as it might/ought fix us up on alacritty, if sixel on alacritty ever works there again (i was sure ayosec/graphics had been merged, but maybe it hasn't yet?).